src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
```

Comment:
You should place this within the component

Reply:
I try to do that but for some reason, the code wouldn't work so I kept it outside the component.